src/main/kotlin/com/hj/leetcode/kotlin/problem445/Solution.kt

```diff
@@ -0,0 +1,48 @@
+package com.hj.leetcode.kotlin.problem445
+
+import com.hj.leetcode.kotlin.common.model.ListNode
+
+/**
+ * LeetCode page: [445. Add Two Numbers II](https://leetcode.com/problems/add-two-numbers-ii/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(M+N) and Space O(M+N) where M and N are the number of nodes in l1 and l2;
+     */
+    fun addTwoNumbers(l1: ListNode?, l2: ListNode?): ListNode? {
+        val digits1 = l1.values()
+        val digits2 = l2.values()
```

Comment:
prefer to construct the stack directly on ListNode instead of its value

Reply:
Revised at https://github.com/hj-core/hj-leetcode-kotlin/pull/107/commits/d03b44b1e02c5319da9bdcf872278b72681aa6c1